README.md

```diff
@@ -142,3 +142,21 @@ HTTPS=true mvn quarkus:dev
 as usual - the `HTTPS=true` will use secured connections on the live-reload proxy on port 3000.
 
 When you want to revert back to plain HTTP, run `./disable-https.sh` and drop the `HTTPS=true` env var.
+
+## SSL in `horreum-client`
+You can set a certificate in runtime for `HorreumClient`
+```java
+InputStream fis = new FileInputStream("/path/to/my-cert.pem");
```

Comment:
Could you rather move all of this to a utility method next to `sslContext(...)`?

Reply:
Done.